src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx

```diff
@@ -756,6 +775,34 @@ mutation {
 }
 ```
   </Collapser>
+
+  <Collapser
+    id="upgrade-monitor-runtime"
+    title="Upgrade a monitor's runtime"
+    >
+  Use this to upgrade a simple browser, scripted API, or scripted browser monitor from using a legacy runtime to the new Chrome 100 or Node 16.10 runtime. See [optional-fields](#optional-fields) for additional details on runtime settings. This example will upgrade the runtime for a scripted browser monitor. Use the `syntheticsUpdateSimpleBrowserMonitor` mutation for simple browser monitors and `syntheticsUpdateScriptApiMonitor` for scripted API monitors.
```

Comment:
```suggestion
You can use the below example to update a simple browser, scripted API, or scripted browser monitor to the new Chrome 100 or Node 16.10 runtimes. The following example upgrades a scripted browser monitor to the new runtime from the legacy runtime. 

* To update a simple browser monitor using the below example, change the mutation to `syntheticsUpdateSimpleBrowserMonitor` instead of `syntheticsUpdateScriptBrowserMonitor`. 
* To update a scripted API monitor using the below example, change the mutation to `syntheticsUpdateScriptApiMonitor` instead of `syntheticsUpdateScriptBrowserMonitor`.

Check out [optional-fields](#optional-fields) for additional details on runtime settings. 
```